homeassistant/components/venstar/sensor.py

```diff
@@ -210,3 +229,16 @@ def native_unit_of_measurement(self) -> str | None:
     value_fn=lambda coordinator, sensor_name: coordinator.runtimes[-1][sensor_name],
     name_fn=lambda coordinator, sensor_name: f"{coordinator.client.name} {RUNTIME_ATTRIBUTES[sensor_name]} Runtime",
 )
+
+INFO_ENTITIES: tuple[VenstarSensorEntityDescription, ...] = (
+    VenstarSensorEntityDescription(
+        key="schedulepart",
+        device_class=SensorDeviceClass.ENUM,
+        options=list(SCHEDULE_PARTS.values()),
+        uom_fn=lambda _: None,
+        value_fn=lambda coordinator, sensor_name: SCHEDULE_PARTS[
+            coordinator.client.get_info(sensor_name)
+        ],
+        name_fn=lambda coordinator, sensor_name: f"{coordinator.client.name} Schedule Part",
```

Comment:
Side note for the future: We recommend existing integrations to move to adhering to our entity naming guidelines and let the device info add the device name to the entity name by default.

https://developers.home-assistant.io/docs/core/entity#entity-naming